dotcom-rendering/src/components/ImageComponent.tsx

```diff
@@ -46,10 +46,6 @@ const timelineBulletStyles = css`
 		top: -6px;
 		left: -6.5px;
 
-		${from.mobileLandscape} {
```

Comment:
Anticipating https://github.com/guardian/dotcom-rendering/pull/11250/files#diff-0c7cc56008f4d46df8e87973dd004c67ca95292c8cda32f52d3b58dbfc3e169dR170 to line up the bullet points